<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EssayDialog</class>
 <widget class="QDialog" name="EssayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>题干：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="descriptionEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>分值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="scoreSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>关键词：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="keywordsEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="ruleGroup">
        <property name="title">
         <string>评分规则</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="containsAnyRadio">
           <property name="text">
            <string>包含任意关键词即得分</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="containsAllRadio">
           <property name="text">
            <string>包含所有关键词得分</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="coutKeywordsRadio">
           <property name="text">
            <string>根据包含关键词的数量按比例得分</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>确定</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>EssayDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>EssayDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
